cap_client/cap_api.py

```diff
@@ -442,3 +443,22 @@ def clone(self, pid):
                                   method='post',
                                   headers={'Content-Type': 'application/json',
                                            'Accept': 'application/basic+json'})
+
+    ##############
+    # REPOS
+    ##############
+
+    def upload_repository(self, pid, url, event_type=None):
+        return self._make_request(url='deposits/{}/actions/upload'.format(pid),
+                                  expected_status_code=201,
+                                  method='post',
+                                  data=json.dumps({
+                                      'pid': pid,
+                                      'url': url,
+                                      'webhook': True if event_type else False,
+                                      'event_type': event_type
```

Comment:
this is not gonna work as expected, if you pass some value other than 'push' or 'release'
try to run to see what's gonna happen ^^
you can copy paste this line from what was suggested in issue descripiton [here](https://github.com/cernanalysispreservation/cap-client/issues/98)